examples/playground/app/websites/runs/[run_id]/page.tsx

```
@@ -0,0 +1,403 @@
+                                </summary>
+                                <div className="mt-2 p-2 bg-muted/50 rounded-md text-xs overflow-auto max-h-40">
+                                  <pre>
+                                    {JSON.stringify(stepTask.output, null, 2)}
+                                  </pre>
+                                </div>
+                              </details>
+                            </div>
+                          )}
+                        </div>
+                      );
+                    })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)